src/main/java/Problem1.java

```diff
@@ -3,6 +3,13 @@
     // Do not change signature (function name, parameters)
     public static int binarySearch(int[] data, int target) {
         // Add your solution
-        return -1; // Place holder
+        int searchValue = 7; //Value to look for
+        int endValue = -1; //Output
+        for (int i = data.length - 1; i >= 0; i--) {
+            if (target == data[i]) {
```

Comment:
-10 linear sort rather than binary sort.

Reply:
using the wrong algorithm will cost you most of the points in later homework. please check with instructor if you are not sure.